### PR TITLE
Feature display seller info

### PR DESCRIPTION
**Pull request by Dustnclay**
# Description
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
- [ ] New feature (non-breaking change which adds functionality)

## Change Status
- [ ] Complete, but not tested (may need new tests)

# Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import axios from 'axios';
 
 // we will define a bunch of API calls here.
 const apiUrl = `${process.env.REACT_APP_API_URI}/profiles`;
+// const apiUrlId = `${process.env.REACT_APP_API_URI}/profiles/${myId}`;
```
Comment (edelveiss) [inline src/api/index.js]: Delete this comment.

File: src/components/sellerPages/profile/dashboard/myInfoSection/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+      <button onClick={clicked}>fetch</button>
+    </>
+  );
+}
+
+const mapStateToProps = state => {
+  return {
+    myInfo: state.information,
+    // myName: state.information.info.seller_name,
```
Comment (edelveiss) [inline src/components/sellerPages/profile/dashboard/myInfoSection/index.js]: Delete all of these comments.

File: src/state/reducers/myInfoReducer.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import {
+  FETCH_MY_INFO_START,
+  FETCH_MY_INFO_SUCCESS,
+  FETCH_MY_INFO_ERROR,
+} from '../actions/index';
+
+const initialState = {
+  seller_name: 'redux',
```
Comment (edelveiss) [inline src/state/reducers/myInfoReducer.js]: You can use something like this:
const initialState = {
  myInfo: {},
  getMyInfoStatus: requestStatus.ready,
};
So we can  check if info is loading. As an example, look at addProductReducer.js .

File: src/state/reducers/myInfoReducer.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  phone_number: 'redux',
+  email_address: 'redux',
+  description: 'redux',
+};
+const myInfoReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_MY_INFO_START:
+      return { ...state };
+    case FETCH_MY_INFO_SUCCESS:
+      //   const newState = action.payload;
```
Comment (edelveiss) [inline src/state/reducers/myInfoReducer.js]: Delete this comment.
Dustin, you have done nice job!

File: .env.sample [hunk 5]
```diff
@@ -1,3 +1,3 @@
-REACT_APP_CLIENT_ID=example
+REACT_APP_CLIENT_ID=0oa18clqjs07fbtS84x7
```
Comment (edelveiss) [inline .env.sample]: leave REACT_APP_CLIENT_ID=example
REACT_APP_OKTA_ISSUER_URI="https://name-438r8hr.okta.com/oauth2/default"

**Review by edelveiss — APPROVED** [review]
Great job, Dustin!

**Review by edelveiss — APPROVED** [review]

**Review by aaronmatson1 — APPROVED** [review]
Good Job! looks good!

**Pull request merged**